client/src/components/Console.tsx

```diff
@@ -95,8 +91,18 @@ export default function Console() {
       fetchDocuments();
       setRefreshNeeded(false);
     }
-  }, [selectedView, refreshNeeded]);
+  }, [refreshNeeded]);
+  useMemo(() => {
+    const entireMunicipalityDocs = tmpDocuments.filter(
```

Comment:
Is it possible to show the record numbers before and after filtering? Just because the user can understand the changes immediately